packages/ui-components/src/components/windowedlist.ts

```diff
@@ -1623,6 +1651,15 @@ export class WindowedLayout extends PanelLayout {
   }
 }
 
+/**
+ * Windowed list model interface
+ */
+export interface ISimpleObservableList {
```

Comment:
It looks like this needs  a generic T:


```suggestion
export interface ISimpleObservableList<T = any> {
```

Though ideally we should be able to constrain any further.

`Error: ../ui-components/src/components/windowedlist.ts(1658,28): error TS2304: Cannot find name 'T'.`